### DISCUSSION TITLE
Perspective in SvelteKit

### DISCUSSION THREAD

**Discussion by riziles** [perspective-dev/perspective#2147] [Show and tell]
Implemented a bare bones Perspective dashboard in SvelteKit with the most excellent [Skeleton UI](www.skeleton.dev):

https://svelte-perspective-starter1.vercel.app/

Next step: full Tailwind integration?

Thank you for creating such an amazing product and for making it open source!

**Comment by texodus** [comment]
Very cool, thanks for sharing @riziles!

**Comment by Khalilsqu** [comment]
would you mind sharing the GitHub repo or the code base? we benefit nothing from the deployed website!

**Reply by sinistersnare** [reply]
There is a github link at the top right of the website.

**Reply by Khalilsqu** [reply]
Thank you very much

**Reply by riziles** [reply]
@Khalilsqu , full disclosure: I just started learning Svelte a few months ago, so it's quite possible I'm not doing things right. But I love Perspective and [Skeleton](https://github.com/skeletonlabs/skeleton), and I've been been using this combo for a few weeks and it seems to be working great. Also, here's a link to the repo for posterity: 

https://github.com/riziles/svelte-perspective-starter1
